nucleus/annotation.py

```diff
@@ -90,7 +90,7 @@ def __str__(self):
     @classmethod
     def from_json(cls, payload: dict):
         return cls(
-            mask_url=payload.get(MASK_URL_KEY),
+            mask_url=str(payload.get(MASK_URL_KEY)),
```

Comment:
Thanks for fixing this typing mistake!

I think we should raise an error if payload[MASK_URL_KEY] is None, instead of converting it to string and passing it on to cause problems elsewhere.

if MASK_URL_KEY not in payload:
  raise ValueError(f"{MASK_URL_KEY} field is required but was missing in the following annotation: {payload}")

Reply:
then you can just do payload[MASK_URL_KEY] and the typing should be ok

Reply:
I address this in #53 so we could just merge that first?